content/blog/ways-to-interact-with-kubernetes-clusters-managed-by-hpe-ezmeral-container-platform.md

```diff
@@ -0,0 +1,180 @@
+
+## WebTerminal
+Inside the Kubernetes Tenant, at the bottom, there is a web terminal for you to interact with Kubernetes using the ```kubectl``` command.
+
+| Screenshot      | Description |
+| ----------- | ----------- |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant.png)      | At the bottom, you can click "Initialize" to initiate the web terminal instance.  |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant-02.png)   | Type your ``` kubectl ``` command to interact with Kubernetes.  |
+
+
+## HPE Kubectl Plugin
+If you want to use kubectl remotely, you must first install the kubectl-plugin to get the session ID to access HPE Ezmeral. To install the kubectl-hpecp plugin, you can run the following command. The HPE kubectl plugin is required to establish the tenant’s authenticated kubectl requests. Therefore, if you want to use kubectl remotely as a tenant user, you must first install the HPE kubectl plugin (kubectl-hpecp) to get an HPE ECP authentication token (a session ID), fetch the kubconfig manifest file, and then interact with the managed K8s clusters.
+
+To install the kubectl-hpecp plugin, first fetch the plugin from the WebUI. Log in as a tenant user, and click on Download HPE Kubectl Plugin (as shown in the picture below) and download the plugin and installation instructions according to your target operating system. The following codes are executed in Linux environment.
+
+| Screenshot      | Description |
+| ----------- | ----------- |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant-03.png)      | You can download the required binary here as well.  |
+
+### Installation of ```kubectl``` and ```kubectl-hpecp```
+#### Step 1: Make sure you have ```kubectl``` install.
+```bash
+# Download the latest version of kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+# And place it anywhere in your PATH:
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+```
+> Reference:
+> - [How to install and set up kubectl on Linux](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+> 
+
+#### Step 2: Install hpe kubectl plugin
+```bash
+# This link might subject to be changed, renew this link on HPE Ezmeral
+# Download kubectl-hpecp binary and untar the file
+wget https://bluedata-releases.s3.amazonaws.com/kubectl-epic/3.4/14/linux/kubectl-hpecp.star
+tar xf kubectl-hpecp.star
+# And place it anywhere in your PATH:
+sudo mv ./kubectl-hpecp /usr/local/bin
+```
+Check kubectl-hpecp is installed correctly.
+
+| Command | Screenshot      | 
+| ----------- | ----------- |
+| ``` kubectl plugin list ``` | ![](https://github.com/helloezmeral/cdn/raw/main/kubectl-plugin-list.png)        |
+| ``` kubectl hpecp -h ``` | ![](https://github.com/helloezmeral/cdn/raw/main/kubectl-hpecp-h.png)       |
+
+> Reference:

```

Comment:
References: (with an 's' as there are more than one reference).
The list items for the references appear in a single line in the rendered blog post.  Please make sure you insert a line break after "References:"